tools/generic_includes.php

```diff
@@ -23,7 +23,7 @@
 $client     = new NDB_Client();
 $client->makeCommandLine();
 $client->initialize($configFile);
-$DB            = Database::singleton();
+$DB            = NDB_Factory::singleton()->database();
```

Comment:
I don't think this has anything to do with this PR?

Reply:
the last commit before my changes dates May 31st a2852be695890ba9932372f03feb56010d5d20b2

It does not look like this is fixed on main: https://github.com/aces/Loris/blob/7ff20e871b54bb188088dfd1af15473823672f79/tools/generic_includes.php#L26

I think that is one that went under the radar as it was in the tools directory. Should I send a PR to just change that and assign it to you?

Reply:
Yes, that would be good. #8074 is what I was thinking of (it went to main..) I might have missed it since it's under tools

Reply:
@driusan done! See https://github.com/aces/Loris/pull/8128